### PR TITLE
404 page added

### LINKED ISSUE #570: Create a 404 page

### PR DESCRIPTION
**Pull request by shivam1192**
As seen in one of the issues I have added a well responsive 404 page. Below added a screenshot of how it looked. 
image

Fixes #570

### CODE REVIEW

**Comment by shivam1192** [comment]
@benparsons Look this page and if there is any changes please suggest.

**Comment by aaronraimist** [comment]
Fixes #570

File: gatsby/static/images/ss.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```
Comment (aaronraimist) [inline gatsby/static/images/ss.sh]: This file can probably be deleted

Comment (shivam1192) [reply gatsby/static/images/ss.sh]: yeah @aaronraimist I just added it mistakenly

**Comment by shivam1192** [comment]
@aaronraimist I created these vectors in adobe illustrator.

**Comment by benparsons** [comment]
@shivam1192 this looks great! I'm thrilled you wanted to contribute this work. Can you do something:

* add a "signed off" line as mentioned in https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.rst#sign-off

**Comment by shivam1192** [comment]
sure @benparsons I have added a signed-off line in my new commit

**Comment by benparsons** [comment]
Let me take this to our design team to talk about, thanks.

**Pull request closed without merging**